client/components/token-field/index.jsx

```diff
@@ -81,7 +91,8 @@ var TokenField = React.createClass( {
 					scrollIntoView={ this.state.selectedSuggestionScroll }
 					isExpanded={ this.state.isActive }
 					onHover={ this._onSuggestionHovered }
-					onSelect={ this._onSuggestionSelected } />
+					onSelect={ this._onSuggestionSelected }
+				/>
```

Comment:
I just had the linter warn me on a branch that the closing tag should be on the same line as the last prop.  This was "new" to me, but figured I'd mention it here too.